unique_toolkit/unique_toolkit/language_model/infos.py

```diff
@@ -20,6 +20,35 @@ class LanguageModelName(StrEnum):
     AZURE_GPT_4o_MINI_2024_0718 = "AZURE_GPT_4o_MINI_2024_0718"
 
 
+class EncoderName(StrEnum):
+    O200K_BASE = "o200k_base"
+    CL100K_BASE = "cl100k_base"
+
+
+def get_encoder_name(model_name: LanguageModelName) -> Optional[EncoderName]:
+    LMN = LanguageModelName
+    match model_name:
+        case (
+            LMN.AZURE_GPT_35_TURBO
+            | LMN.AZURE_GPT_35_TURBO_16K
```

Comment:
Would or be better instead of `|`?

Reply:
`or` gives me a syntax error 😮

Reply:
Maybe it is not implement for the match flow

Reply:
Ah, my bad then. I think this looks good.